src/index.html

```diff
@@ -6,9 +6,36 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img src="images/logo.png" alt="moyo-logo"
+      class="logo">
+      <nav>
+        <ul class="menu">
+          <li class="link button is-active"> <a href="#" class="nav_link is-active">Apple</a></li>
+          <li class="link button">Samsung</li>
+          <li class="link button">Smartphones</li>
+          <li class="link button" data-qa="hover">Laptops & Computers</li>
+          <li class="link button">Gadgets</li>
+          <li class="link button">Tablets</li>
+          <li class="link button">Photo</li>
+          <li class="link ">Video</li>
+        </ul>
```

Comment:
You need to fix it everywhere
<img width="1020" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/a85c53cf-b790-41cc-b244-6a9186f6c715">